101-vm-ws2016-visualstudio2019/GettingStarted-wsvs.md

```diff
@@ -0,0 +1,195 @@
+# Simple Windows Server 2016 - Datacenter with Visual Studio 2019
+
+The intent of this README is to guide you in a deployment a **Windows Server 2016 - Datacenter Standard-D2-v2** (Two VCPUs and 7 GIB RAM) with the **Visual Studio Community 2019**.  
+
+This template was designed to be an easy and fast way to create a Virtual Machine ready for you to start coding with Visual Studio.  
+
+You can deploy this template either using the [Azure Portal](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy-portal) or [Azure CLI](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy-cli). Let's dig on the template and have some fun.
+
+###The Template
+Don't let the size of the template scares you. The structure is very intuitive and once that you get the gist of it, you gonna see how easier your life will be regarding creating resources on Azure.
+
+The only parameters that we need to inform are:  **adminUsername**, **adminPassword** and **vmName**.
+
+Don't worry about changing anything on the file, either on the portal or using Azure CLI, you gonna be asked to insert this information, but bear in mind that there is some requirement for those two parameters:
+
+
+- *adminUsername:* Usernames can be a maximum of 20 characters in length and cannot end in a period (".").
+
+The following usernames are not allowed:  
+
+  | | ||
```

Comment:
this does not render correctly in github

Reply:
Also changed that.